### PR TITLE
Next

### PR DESCRIPTION
**Pull request by adamgruber**
Tracking `next` branch changes

### CODE REVIEW

File: src/bin/types.js [hunk 1]
```diff
@@ -1,13 +1,29 @@
 const t = require('tcomb');
 const { isUUID, isISO8601 } = require('validator');
 
-const PercentClass = t.enums.of([ 'success', 'warning', 'danger' ], 'PercentClass');
```
Comment (jimberley) [inline src/bin/types.js]: These changes are going to break Cypress-generated Mocha reports. You may get a flood of grumpy Cypress users inbound. Just FYI.

This change is a regression of this earlier change:

https://github.com/adamgruber/mochawesome-report-generator/issues/111

Comment (adamgruber) [reply src/bin/types.js]: I'm not entirely sure what change you're referring to. The changes made as a result of #111 still exist in the latest version. https://github.com/adamgruber/mochawesome-report-generator/blob/master/src/bin/types.js#L4

Comment (jimberley) [reply src/bin/types.js]: I was referring to the removal of `warning` and `danger` from the `PercentClass` const. I must've mis-linked the issue.

Cypress produces `mochawesome.json` files which contain `warning` and `danger` values in the `PercentClass` key. I believe that their removal in this PR might make creating tests from Cypress-created `mochawesome.json` files impossible if the test output includes either of those values.

Comment (adamgruber) [reply src/bin/types.js]: > Cypress produces mochawesome.json files

Unless I'm mistaken, this is only created when using `mochawesome` as a reporter. The changes to `PercentClass` here are in-line with changes made to `mochawesome` in `4.0.0`. It was a deliberate breaking change. If you upgrade to the latest `mochawesome`, it requires the latest `mochawesome-report-generator` so there shouldn't be any mismatch issues.

Comment (jimberley) [reply src/bin/types.js]: Awesome. Thank you for the clarification, Adam. I appreciate the time (and the tool, BTW).

I'll try upgrading both and see how that goes (and take up the remainder with Cypress).

**Pull request merged**